### PR TITLE
Created Follina Vulnerability readme file.

### PR DESCRIPTION
**Pull request by Tholkappiar**
Hello, Maintainers!

I hope you are doing well. I've submitted a pull request by creating a Readme file about Follina Vulnerability.
I wanted to let you know that I am open to making any necessary changes or improvements according to your suggestions. Please feel free to review my changes and provide any feedback you may have.

Thank you for your time and consideration. I look forward to hearing from you soon.

Best regards,
Tholkappiar

### CODE REVIEW

File: pages/vulnerabilities/Follina_vulnerability.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+
+layout: col-sidebar
+title: "The Follina Vulnerability: A Critical Threat to Microsoft Office"
+author: "Tholkappiar"
+contributors: ["Tholkappiar M"]
+permalink: /The_Follina_Vulnerability
+tags: ["attack", "Follina", "Microsoft"]
```
Comment (kingthorin) [inline pages/vulnerabilities/Follina_vulnerability.md:8]:
```suggestion
tags: ["vulnerability", "Follina", "Microsoft"]
```

File: pages/vulnerabilities/Follina_vulnerability.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+
+layout: col-sidebar
+title: "The Follina Vulnerability: A Critical Threat to Microsoft Office"
+author: "Tholkappiar"
+contributors: ["Tholkappiar M"]
```
Comment (kingthorin) [inline pages/vulnerabilities/Follina_vulnerability.md:6]:
```suggestion
contributors: []
```

File: pages/vulnerabilities/Follina_vulnerability.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Key Details
+
+- **CVE-2022-30190**: Officially recognized as [CVE-2022-30190](https://nvd.nist.gov/vuln/detail/CVE-2022-30190) by the National Institute of Standards and Technology (NIST), Follina warrants tracking due to its severity.
+- **Phishing Campaigns**: Cybercriminals actively exploit Follina through sophisticated phishing campaigns, luring users into opening malicious Office documents or links that trigger the vulnerability.
+- **MSDT Protocol**: The critical issue lies in the manipulation of the "Microsoft Support Diagnostic Tool" (MSDT) protocol. Attackers leverage this protocol to execute their own PowerShell commands, often without user interaction.
+- **Diverse Attack Vectors**: Follina can strike via email-delivered malicious Office documents, USB devices, or even during file previews (e.g., .rtf formats).
+- **Discovery Timeline**: Unveiled as a zero-day vulnerability on `May 27, 2022`, the first known malware exploiting it surfaced on `April 7, 2022`, suggesting prior exploitation.
+- **Escalation of Phishing Campaigns**: With Follina's discovery, cybersecurity experts have noted a surge in phishing campaigns employing this vulnerability.
```
Comment (kingthorin) [inline pages/vulnerabilities/Follina_vulnerability.md:36]:
```suggestion
- **Escalation of Phishing Campaigns**: With Follina's discovery, cybersecurity experts noted a surge in phishing campaigns employing this vulnerability.
```

File: pages/vulnerabilities/Follina_vulnerability.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+## Protecting Against Follina
+
+Addressing Follina necessitates a multi-pronged approach:
+
+- **Patch and Update**: Swiftly apply Microsoft's security updates to shield against Follina.
+- **Awareness is Key**: Educate staff about phishing threats, fostering a vigilant user base.
+- **Disabling MSDT**: Initially, Microsoft recommended disabling the MSDT protocol in Windows to mitigate attack vectors.
+- **Advanced Cybersecurity**: Invest in advanced cybersecurity solutions for robust defense against zero-day attacks.
+
+
```
Comment (kingthorin) [inline pages/vulnerabilities/Follina_vulnerability.md:70]:
```suggestion
```

File: pages/vulnerabilities/Follina_vulnerability.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+- [Detecting Follina (CVE-2022-30190) with Indicators of Compromise (IOCs) and AI Engine (AIE) trend rules to highlight anomalous activity | LogRhythm](https://www.logpoint.com/en/blog/detecting-follina-microsoft-office-remote-code-execution-zero-day/)
+
+- [Follina — Microsoft Office code execution vulnerability - Infosec Resources](https://www.bleepingcomputer.com/news/security/windows-zero-day-exploited-in-us-local-govt-phishing-attacks/)
+
+- [Mitigating the Follina Zero-Day Vulnerability (CVE 2022-30190) with Privilege Management for Windows - BeyondTrust](https://www.beyondtrust.com/blog/entry/mitigating-the-follina-zero-day-vulnerability-cve-2022-30190-with-privilege-management-for-windows)
+
+- [Detect the Follina MSDT Vulnerability (CVE-2022-30190) with Qualys Multi-Vector EDR & Context XDR](https://vulners.com/qualysblog/QUALYSBLOG:A63B251EBA1A69DBCD57674990704F6C)
+
+- [Follina Vulnerability - BlackBerry](https://www.blackberry.com/us/en/solutions/endpoint-security/security-vulnerabilities/follina-vulnerability)
```
Comment (kingthorin) [inline pages/vulnerabilities/Follina_vulnerability.md:83]:
```suggestion
- [Follina (CVE-2022-30190): Microsoft Office Zero-Day Exploit | LogRhythm](https://www.huntress.com/blog/microsoft-office-remote-code-execution-follina-msdt-bug)
- [Detecting Follina (CVE-2022-30190) with Indicators of Compromise (IOCs) and AI Engine (AIE) trend rules to highlight anomalous activity | LogRhythm](https://www.logpoint.com/en/blog/detecting-follina-microsoft-office-remote-code-execution-zero-day/)
- [Follina — Microsoft Office code execution vulnerability - Infosec Resources](https://www.bleepingcomputer.com/news/security/windows-zero-day-exploited-in-us-local-govt-phishing-attacks/)
- [Mitigating the Follina Zero-Day Vulnerability (CVE 2022-30190) with Privilege Management for Windows - BeyondTrust](https://www.beyondtrust.com/blog/entry/mitigating-the-follina-zero-day-vulnerability-cve-2022-30190-with-privilege-management-for-windows)
- [Detect the Follina MSDT Vulnerability (CVE-2022-30190) with Qualys Multi-Vector EDR & Context XDR](https://vulners.com/qualysblog/QUALYSBLOG:A63B251EBA1A69DBCD57674990704F6C)
- [Follina Vulnerability - BlackBerry](https://www.blackberry.com/us/en/solutions/endpoint-security/security-vulnerabilities/follina-vulnerability)
```

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
https://owasp.org/www-community/vulnerabilities/follina

**Comment by Tholkappiar** [comment]
Thank You ! Thank you for your time and consideration

**Pull request merged**